zipline/data/data_portal.py

```diff
@@ -234,6 +212,16 @@ def _get_ctable(self, asset):
 
         return bcolz.open(path, mode='r')
 
+    def get_last_traded_dt(self, asset, dt, data_frequency):
+        """
+        Given an asset and dt, returns the last traded dt from the viewpoint
+        of the given dt.
+
+        If there is a trade on the dt, the answer is dt provided.
+        """
+        if data_frequency == 'minute':
+            return self._equity_minute_reader.get_last_traded_dt(asset, dt)
+
```

Comment:
Looking at this again, with wiring this to SidView, we'll have to implement for daily.